e2e test failure (`.status:` key missing) due to bad controller image tag

This was running locally on [a commit](https://github.com/kudobuilder/kudo/commit/b85bd6c6201ced8dc546115e2a07c5a35c221b5f) which is marked as green in GitHub.
```
+ ./bin/kubectl-kudo test --config kudo-e2e-test.yaml
Running E2E tests without junit output
=== RUN   kudo
2020/01/09 10:08:59 CustomResourceDefinition:/instances.kudo.dev created
2020/01/09 10:08:59 CustomResourceDefinition:/operators.kudo.dev created
2020/01/09 10:08:59 CustomResourceDefinition:/operatorversions.kudo.dev created
2020/01/09 10:08:59 CustomResourceDefinition:/teststeps.kudo.dev created
2020/01/09 10:08:59 CustomResourceDefinition:/testsuites.kudo.dev created
2020/01/09 10:08:59 Operator:default/configmap-operator created
2020/01/09 10:08:59 OperatorVersion:default/configmap-operator-1.0 created
2020/01/09 10:08:59 CustomResourceDefinition:/operators.kudo.dev updated
2020/01/09 10:08:59 CustomResourceDefinition:/operatorversions.kudo.dev updated
2020/01/09 10:08:59 CustomResourceDefinition:/instances.kudo.dev updated
2020/01/09 10:08:59 Namespace:/kudo-system created
2020/01/09 10:09:00 ServiceAccount:kudo-system/kudo-manager created
2020/01/09 10:09:00 ClusterRoleBinding:/kudo-manager-rolebinding created
2020/01/09 10:09:00 Service:kudo-system/kudo-controller-manager-service created
2020/01/09 10:09:01 StatefulSet:kudo-system/kudo-controller-manager created
2020/01/09 10:09:01 Operator:default/test-operator created
2020/01/09 10:09:02 OperatorVersion:default/test-operator-1.0 created
2020/01/09 10:09:02 Operator:default/toy created
2020/01/09 10:09:02 OperatorVersion:default/toy-v1 created
=== RUN   kudo/harness
=== RUN   kudo/harness/cli-install-uninstall
=== PAUSE kudo/harness/cli-install-uninstall
=== CONT  kudo/harness/cli-install-uninstall
--- FAIL: kudo (336.32s)
    harness.go:217: Running tests with KIND.
    harness.go:67: Going to run test suite with timeout of 300 seconds for each step
    --- FAIL: kudo/harness (0.01s)
        --- FAIL: kudo/harness/cli-install-uninstall (302.82s)
            logger.go:37: 10:09:02 | cli-install-uninstall | Ignoring first-operator as it does not match file name regexp: ^(\d+)-([^.]+)(.yaml)?$
            logger.go:37: 10:09:02 | cli-install-uninstall | Creating namespace: kudo-test-apparent-guinea
            logger.go:37: 10:09:02 | cli-install-uninstall/0-install | starting test step 0-install
            logger.go:37: 10:09:02 | cli-install-uninstall/0-install | Running command: kubectl {kudo install --instance first-operator ./first-operator %!s(bool=true) %!s(bool=false)}
            logger.go:37: 10:09:02 | cli-install-uninstall/0-install | 
            logger.go:37: 10:09:02 | cli-install-uninstall/0-install | operator.kudo.dev/v1beta1/first-operator created
                operatorversion.kudo.dev/v1beta1/first-operator-0.1.0 created
                instance.kudo.dev/v1beta1/first-operator created
                
            logger.go:37: 10:14:05 | cli-install-uninstall/0-install | test step failed 0-install
            case.go:155: --- Instance:kudo-test-apparent-guinea/first-operator
                +++ Instance:kudo-test-apparent-guinea/first-operator
                @@ -1,9 +1,11 @@
                 apiVersion: kudo.dev/v1beta1
                 kind: Instance
                 metadata:
                +  labels:
                +    kudo.dev/operator: first-operator
                   name: first-operator
                   namespace: kudo-test-apparent-guinea
                -status:
                -  aggregatedStatus:
                -    status: COMPLETE
                +spec:
                +  operatorVersion:
                +    name: first-operator-0.1.0
                 
                
            case.go:155: resource Instance:kudo-test-apparent-guinea/first-operator: .status: key is missing from map
            case.go:155: deployments.apps "nginx-deployment" not found
            case.go:158: failed in step 0-install
            logger.go:37: 10:14:05 | cli-install-uninstall | cli-install-uninstall events from ns kudo-test-apparent-guinea:
            logger.go:37: 10:14:05 | cli-install-uninstall | Deleting namespace: kudo-test-apparent-guinea
    harness.go:442: collecting cluster logs to kind-logs-1578561245
    harness.go:469: tearing down kind cluster
FAIL
```

--- a/hack/run-e2e-tests.sh
+++ b/hack/run-e2e-tests.sh
@@ -13,7 +13,7 @@ docker build . \
     -t "kudobuilder/controller:$VERSION"
 
 # Generate the kudo.yaml that is used to install KUDO while running e2e-test
-./bin/kubectl-kudo init --dry-run --output yaml --kudo-version kudobuilder/controller:$VERSION \
+./bin/kubectl-kudo init --dry-run --output yaml --kudo-image kudobuilder/controller:$VERSION \
     | sed -E -e '/imagePullPolicy/ s/Always/Never/' \
     > test/manifests/kudo.yaml
 

--- a/hack/run-e2e-tests.sh
+++ b/hack/run-e2e-tests.sh
@@ -13,8 +13,8 @@ docker build . \
     -t "kudobuilder/controller:$VERSION"
 
 # Generate the kudo.yaml that is used to install KUDO while running e2e-test
-./bin/kubectl-kudo init --dry-run --output yaml \
-    | sed -E -e "/image:/ s/:[[:alnum:].]+/:$VERSION/" -e '/imagePullPolicy/ s/Always/Never/' \
+./bin/kubectl-kudo init --dry-run --output yaml --kudo-version kudobuilder/controller:$VERSION \
+    | sed -E -e '/imagePullPolicy/ s/Always/Never/' \
     > test/manifests/kudo.yaml
 
 sed "s/%version%/$VERSION/" kudo-e2e-test.yaml.tmpl > kudo-e2e-test.yaml

--- a/hack/run-e2e-tests.sh
+++ b/hack/run-e2e-tests.sh
@@ -14,7 +14,7 @@ docker build . \
 
 # Generate the kudo.yaml that is used to install KUDO while running e2e-test
 ./bin/kubectl-kudo init --dry-run --output yaml \
-    | sed -E -e "/image:/ s/:[[:alnum:]]+/:$VERSION/" -e '/imagePullPolicy/ s/Always/Never/' \
+    | sed -E -e "/image:/ s/:[[:alnum:].]+/:$VERSION/" -e '/imagePullPolicy/ s/Always/Never/' \
     > test/manifests/kudo.yaml
 
 sed "s/%version%/$VERSION/" kudo-e2e-test.yaml.tmpl > kudo-e2e-test.yaml

